firedrake/slate/slate.py

```diff
@@ -549,38 +590,40 @@ def __init__(self, tensor, coefficient):
             "coefficient function space."
         )
         super(Action, self).__init__()
-        self.tensor = tensor
-        self._acting_coefficient = coefficient
+        self.operands = tensor, coefficient
```

Comment:
I think so, following the gem model, `operands` are closed over gem objects and "non-child-data" don't live on `operands`.

I prefer `actee` to `acting_coefficient` (since the coefficient isn't acting, the tensor is...)